### PR TITLE
Weird Item Categorization Fixes Part 1

### PR DESCRIPTION
**Pull request by Theundyingcode**
#### Summary

```SUMMARY: Bugfixes "Fix weird item categorization part 1"```

#### Purpose of change

In reference to #26221

#### Describe the solution

Made the following category changes with the goal of consistency.  To that end, almost all wearable tools are CLOTHING so change others to match and if I readily recognize it as food, it's food.

Item | Original Category | New Category
-- | --- | -- 
heavy stick | WEAPONS | SPARE PARTS
long stick | WEAPONS | SPARE PARTS
chainsaw lajatang (on) | TOOLS | WEAPONS
chainsaw lajatang (off) | TOOLS | WEAPONS
electric chainsaw lajatang (on) | TOOLS | WEAPONS
electric chainsaw lajatang (off) | TOOLS | WEAPONS
XL gas mask | TOOLS | CLOTHING
gas mask | TOOLS | CLOTHING
game watch | TOOLS | CLOTHING
chainmail armor | CLOTHING | ARMOR
chainmail coif | CLOTHING | ARMOR
chainmail leggings | CLOTHING | ARMOR
chainmail vest | CLOTHING | ARMOR
pair of chainmail sleeves | CLOTHING | ARMOR
baseball helmet | CLOTHING | ARMOR
hooded hard hat | CLOTHING | ARMOR
pair of ballistic glasses | CLOTHING | ARMOR
paper arm guards | CLOTHING | ARMOR
pickelhauben | CLOTHING | ARMOR
skid lid | CLOTHING | ARMOR
basic chemistry set | OTHER | TOOLS
almond | OTHER | FOOD
chestnut | OTHER | FOOD
hazelnut | OTHER | FOOD
hickory nuts | OTHER | FOOD
peanut | OTHER | FOOD
pecan | OTHER | FOOD
pistachio | OTHER | FOOD
rock salt | OTHER | FOOD
walnut | OTHER | FOOD

#### Additional context

I went through the enter "Wish for an item menu" with all of the default content mods and made three lists:
1. Items that should get recategorized
2. Items with sketchy categorization or categorization that I don't agree with but want to ask about first
3. Items I have questions about or seem to have other errors

This PR fixes around half of list 1. I plan to open an issue regarding list 2 and deal with list 3 on a case by case basis (probably on the forums)

### CODE REVIEW

File: data/json/items/chemicals_and_resources.json [hunk 1]
```diff
@@ -137,6 +137,7 @@
     "type" : "GENERIC",
     "price_postapoc": 5000,
     "id" : "material_rocksalt",
+    "category" : "food"
```
Comment (mlangsdorf) [inline data/json/items/chemicals_and_resources.json]: missing a comma, should be
    "category" : "food",

File: data/json/items/tools.json [hunk 2]
```diff
@@ -2426,6 +2429,7 @@
   {
     "id": "elec_chainsaw_off",
     "type": "TOOL",
+    "category": "weapon",
```
Comment (mlangsdorf) [inline data/json/items/tools.json]: I use chainsaws more for cutting down trees than for cutting down zombies.

Comment (Theundyingcode) [reply data/json/items/tools.json]: oops, that was a mistake, I got a little too happy when fixing the lajatangs.

**Comment by Rivet-the-Zombie** [comment]
If you're doing food things, could we *please* remove animal poo from the food category?

**Comment by Theundyingcode** [comment]
> If you're doing food things, could we please remove animal poo from the food category?

It's on my list. I'll do it in part 2

File: data/json/items/tools.json [hunk 3]
```diff
@@ -2053,6 +2054,7 @@
   {
     "id": "cs_lajatang_off",
     "type": "TOOL",
+    "category": "weapon",
```
Comment (ZhilkinSerg) [inline data/json/items/tools.json]: We do not have category named `weapon`. It is called `weapons`.

image

File: data/json/items/tools.json [hunk 4]
```diff
@@ -2075,6 +2077,7 @@
   {
     "id": "cs_lajatang_on",
     "type": "TOOL",
+    "category": "weapon",
```
Comment (ZhilkinSerg) [inline data/json/items/tools.json]: Same here.

**Comment by Mecares** [comment]
"id":
"cs_lajatang_off",
"type": "TOOL",
"category": "weapon",
"category": "weapons",
"name": "chainsaw lajatang (off)",
"name_plural": "chainsaw lajatangs (off)",
"description": "A long wooden pole with chainsaws impractically attached to both ends. The product of either genius or insanity, but not both; the weight ensures that only someone both strong and very skilled could possibly hope to use it.",
"weight": 14254,
"volume": 30,
"price": 40000,
"to_hit": -5,
"bashing": 20,
"material": [ "plastic", "steel" ],
"symbol": "/",
"color": "red",
"ammo": "gasoline",
"max_charges": 800,
"turns_per_charge": 1,
"revert_to": "chainsaw_off",
"techniques": [ "WBLOCK_1", "SPIN", "SWEEP" ],
"use_action": "CS_LAJATANG_OFF",
"flags": [ "NONCONDUCTIVE" ]

That looks like the turned off chainsaw lajatang still uses gasoline and turns into a chainsaw(off) when empty, or am i reading this wrong?

**Pull request merged**